### PR TITLE
Add a pistol disguised as an appraisal tool

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
This weapon is identical to the Viper but with some small changes. It's only available to logistics workers

## Why / Balance
The gun occupies only a 1x2 slot to mimic the appraisal tool, it can appraise stuff too. The main difference with the viper is that it shoots at 4.5 instead of the Viper's 6.0 bullets per second.
On the why add this gun? Because it's fun to have more hidden weaponry in my opinion even if you can tell by shift clicking as long as it passes a quick check from sec it's fine.

## Technical details
Added a new weapon `WeaponPistolViperAppraisal`, added to the uplink including locales + modified the sprite for the appraise tool.

## Media

https://github.com/user-attachments/assets/74bb9d68-fef1-414b-94ba-29744f7b053d

image
image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
None.

**Changelog**

:cl:
- add: Added a new pistol, a modified Viper meant to look like a regular appraisal tool, it shoots slightly slower than a regular Viper and it's only available to logistics traitors.

### CODE REVIEW

**Comment by deltanedas** [comment]
time for every sec to spam E on every appraisal tool :trollface:

**Comment by Radezolid** [comment]
They can just shift click it, didn't find a way to hide the bullet counter.

**Review by Lyndomen — DISMISSED** [review]

File: Resources/Textures/DeltaV/Objects/Weapons/Guns/Pistols/appraisal-gun.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        },
+        {
+            "name": "mag-0"
+        },
+        {
+            "name": "equipped-BELT",
+            "directions": 4
+        }
+    ],
+    "version": 1
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Objects/Weapons/Guns/Pistols/appraisal-gun.rsi/meta.json]: version at the top

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by therealDLondon** [comment]
Even with the slower firerate I think it should cost slightly more TC than the base viper, as opposed to cheaper

**Comment by Radezolid** [comment]
It cost the same as a viper with the same chance and reduction of TC

**Review by deltanedas — APPROVED** [review]

**Pull request merged**